VocaDbWeb.Core/Models/Tag/TagEditViewModel.cs

```diff
@@ -0,0 +1,150 @@
+#nullable disable
+
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using VocaDb.Model;
+using VocaDb.Model.DataContracts;
+using VocaDb.Model.DataContracts.Globalization;
+using VocaDb.Model.DataContracts.Tags;
+using VocaDb.Model.Domain;
+using VocaDb.Model.Domain.Globalization;
+using VocaDb.Model.Domain.Security;
+using VocaDb.Model.Domain.Tags;
+using VocaDb.Web.Code;
+using VocaDb.Web.Code.Exceptions;
+using VocaDb.Web.Helpers;
+
+namespace VocaDb.Web.Models.Tag
+{
+	// TODO: implement [PropertyModelBinder]

```

Comment:
We no longer need this anymore?

Reply:
You mean PropertyModelBinderAttribute? I think the purpose was to activate FromJsonAttributes. But yeah, might be that in ASP.NET Core it's not needed anymore.